k-distribution/tests/regression-new/set-symbolic-tests/failing/diff-11-spec.k.out.actual

```diff
@@ -0,0 +1,9 @@
+  <k>
+    diffResult ( SetItem ( Y:MyId ) -Set SetItem ( X:MyId ) ) ~> .
```

Comment:
This is very bizarre - somehow we are simplifying `SetItem(Y) -Set (S SetItem(X))` to `(SetItem(Y) -Set SetItem(X)` (removing the `S`), but that is *wrong*, `Y` could be present in `S`. 
_This happens regardless of whether the new simplification rules are imported or not!_
Same for `diff-12` and `diff-13` tests (also `diff-15`, not included here since not relevant with the reduced rules)